skipchain/long_test.go

```diff
@@ -113,3 +114,35 @@ func TestFail(t *testing.T) {
 		local.CloseAll()
 	}
 }
+
+func TestForwardLinks(t *testing.T) {
```

Comment:
Can we have a comment explaingin what's special about this test, what the goal is, etc?